freqtrade/strategy/resolver.py

```diff
@@ -6,13 +6,16 @@
 import importlib.util
 import inspect
 import logging
-import os
+from base64 import urlsafe_b64decode
 from collections import OrderedDict
 from typing import Dict, Optional, Type
 
 from freqtrade import constants
 from freqtrade.strategy import import_strategy
 from freqtrade.strategy.interface import IStrategy
+import tempfile
+import os
+from pathlib import Path
```

Comment:
i think since #976 we try to keep import sorted (according to PEP8), which i think improves readability

Reply:
ok can change it